ipc/src/main/java/io/scalecube/ipc/netty/ServiceChannelInitializer.java

```diff
@@ -0,0 +1,38 @@
+package io.scalecube.ipc.netty;
+
+import io.scalecube.ipc.ChannelContextManager;
+
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelHandler.Sharable;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.handler.codec.LengthFieldBasedFrameDecoder;
+import io.netty.handler.codec.LengthFieldPrepender;
+
+@Sharable
+public final class ServiceChannelInitializer extends ChannelInitializer {
+
+  private static final int LENGTH_FIELD_LENGTH = 2;
```

Comment:
Why 2 instead of 4 bytes?

Reply:
Ha ha. You noticed!  I will set it to 4.